ci-operator/config/redhat-openshift-ecosystem/msp/redhat-openshift-ecosystem-msp-ocp-4.8.yaml

```diff
@@ -19,6 +19,8 @@ tests:
     product: ocp
     timeout: 1h0m0s
     version: "4.8"
+    multi-az: "false"
```

Comment:
The new labels will also need to be added to the `cluster_claim` section of the  `small-single-az-amd64-aws` job in this same file.

Reply:
checking

Reply:
done @dirgim